src/components/Dropdown/DropdownPills.less

```
@@ -0,0 +1,11 @@
+.o-multiselect_choices.pills.bottom {
+  li:first-child {
+    margin-top: 0.625rem;
+    margin-bottom: 0;
+  }
+
+  li {
+    margin-top: 0.5rem;
+    margin-bottom: 0;
+  }
+}
```

To mirror the syntax style of the [design system multiselect](https://github.com/cfpb/design-system/blob/main/packages/cfpb-forms/src/organisms/multiselect.less), you could turn these rules into a modifier class:

```suggestion
@import (reference) '@cfpb/cfpb-design-system';

.o-multiselect_choices__bottom {
  li {
    margin-top: unit((8px / @base-font-size-px), em);
    margin-bottom: 0;

    &:first-child {
      margin-top: unit((10px / @base-font-size-px), em);
    }
  }
}
```